log/brimapi/logger.go

```diff
@@ -60,3 +73,22 @@ func (lh *LogHook) Fire(entry *logrus.Entry) error {
 	}
 	return discardBody(resp)
 }
+
+// Discovery service URI for service:// protocol
+func (lh *LogHook) discoveryServiceURI(URI string) (*url.URL, error) {
```

Comment:
Method name shouldn't be noun, I can't tell what the method does just by looking at it's signature.